apps/api/src/workspace/managers/image.manager.ts

```diff
@@ -50,10 +52,17 @@ export class ImageManager {
     private readonly nodeApiFactory: NodeApiFactory,
     private readonly redisLockProvider: RedisLockProvider,
     private readonly organizationService: OrganizationService,
-  ) {}
+    private readonly configService: TypedConfigService,
+  ) {
+    this.isMaintenanceMode = this.configService.get('maintananceMode')
+  }
 
   @Cron(CronExpression.EVERY_5_SECONDS)
```

Comment:
I believe we can easily stop all cron jobs like this:

```typescript
function stopAllCronJobs(app: INestApplicationContext) {
    const schedulerRegistry = app.get(SchedulerRegistry);
    for (let cronName of schedulerRegistry.getCronJobs().keys()) {
        schedulerRegistry.deleteCronJob(cronName);
    }
}

const app: INestApplicationContext = await NestFactory.createApplicationContext(AppModule);
stopAllCronJobs(app);
```

Ref: https://stackoverflow.com/a/78997341

It'll make them more maintainable.